analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -674,6 +678,26 @@ def __collect_clang_compile_opts(flag_iterator, details):
         return True
 
 
+def __collect_transform_clang_xclang_opts(flag_iterator, details):
+    """Some specific -Xclang constucts needs to be filtered out.
+
+       To generate the proper plist reports and not an LLVM IR
+       as an output the flags needs to be removed.
```

Comment:
as an output the flags needs to be removed. -> as an output, the flags need to be removed.